common.php

```diff
@@ -28,6 +28,16 @@
     );
 }
 
+// The base url for single-sign-on
+$ssoBaseurl = 'http://localhost/';
+$sso = array(
+  'host' => getenv('DATABASE_HOST') ? getenv('DATABASE_HOST') : '127.0.0.1',
+  'port' => getenv('DATABASE_PORT') ? getenv('DATABASE_PORT') : '3306',
+  'dbname' => getenv('DATABASE_NAME') ? getenv('DATABASE_NAME') : 'openeyes',
+  'username' => getenv('DATABASE_USER') ? getenv('DATABASE_USER') : 'openeyes',
+  'password' => getenv('DATABASE_PASS') ? getenv('DATABASE_PASS') : 'openeyes',
```

Comment:
You should be able to shrink the length of these lines by using the ?: operator. It will mean you won't need to repeat the getenv() call for each environment variable.

Reply:
Sorry, yes, I've discussed with @ZheWangJames and @piyushgupta95 and they are doing it through the main repo now.

Reply:
@apigram Hey this settings repo is great. The setting across different installations will drift over time and form a tree. Something like this change looks like it should start at master and go to all the branches that get single sign on. Others may start further up. Good to see everyone plugging away at this :)